pkgs/by-name/ru/rubberband/package.nix

```
@@ -4,11 +4,11 @@
 
 stdenv.mkDerivation rec {
   pname = "rubberband";
-  version = "3.3.0";
+  version = "4.0.0";
 
   src = fetchurl {
     url = "https://breakfastquay.com/files/releases/${pname}-${version}.tar.bz2";
```

```suggestion
    url = "https://breakfastquay.com/files/releases/rubberband-${version}.tar.bz2";
```
https://github.com/NixOS/nixpkgs/issues/277994